zetaforge/forge_runner.py

```diff
@@ -366,8 +364,10 @@ def create_config_json(s2_path, context, registry_port=5000):
         "Database":"./zetaforge.db",
         "KubeContext": context,
         "Local": {
-            "BucketPort":"8333",
-            "RegistryPort": str(registry_port)
+            "BucketPort": 8333,
+            "RegistryPort": registry_port,
+            "RegistryDomain":"example.org",
```

Comment:
For minikube, do we need to set the domain during setup? It's called by the kaniko build line:
```
destination = fmt.Sprintf("%s/%s:%s", cfg.Local.RegistryDomain, block.Action.Container.Image, block.Action.Container.Version)
```